test/psa_serviceaccount_test.go

```diff
@@ -24,39 +24,6 @@ var _ = Describe("GIVEN pod security admission", func() {
 		Expect(err).NotTo(HaveOccurred())
 	})
 
-	Context("WHEN psp is NOT being bypassed", func() {
```

Comment:
These psp tests are no longer needed as we don't have a restricted PSP to test against